.travis.yml

```diff
@@ -0,0 +1,6 @@
+python:
```

Comment:
Add `language: python` at the top to make Travis work better

[Hint](https://github.com/JeffreyMFarley/hew/blob/master/.travis.yml)